librz/il/definitions/bool.c

```diff
@@ -0,0 +1,34 @@
+#include "bool.h"
+
+Bool rz_il_new_bool(bool true_or_false) {
```

Comment:
Why bitvectors functions are `RZ_API` but those are not? Please make them consistent.

Reply:
Ok